fgrpc/grpcrunner_test.go

```diff
@@ -55,9 +55,53 @@ func TestGRPCRunner(t *testing.T) {
 	}
 }
 
+//TODO : find something that actually fails based on MAXSTREAM
+func TestGRPCRunnerMaxStreams(t *testing.T) {
```

Comment:
It would be nice to see this as a table driven test.

Reply:
which table though ? there isn't much more than 1 thing to test